docs/quickstart/sdks.mdx

```diff
@@ -1,11 +0,0 @@
----
-id: sdks
```

Comment:
this was an accident I assume, can you revert?

Reply:
Wans't  sure if that was on accident, removed it